framework/decode/vulkan_ascii_consumer_base.h

```diff
@@ -109,22 +111,52 @@ class VulkanAsciiConsumerBase : public VulkanConsumer
 
   protected:
     template <typename ToStringFunctionType>
-    inline void WriteApiCallToFile(const ApiCallInfo&   call_info,
-                                   const std::string&   functionName,
-                                   util::ToStringFlags  toStringFlags,
-                                   uint32_t&            tabCount,
-                                   uint32_t             tabSize,
-                                   ToStringFunctionType toStringFunction)
+    inline void WriteApiCallToFile(const ApiCallInfo& call_info,
+                                   const std::string& functionName,
+                                   /// @todo Remove toStringFlags, tabCount, tabSize
```

Comment:
I want to keep this PR and its commits minimal so I'm deferring this change to a follow-up PR or else there'd be lines changed all over the place burying the interesting parts of the PR. From a practical point of view, this function is inlined and so we won't pay for passing these unused parameters in release builds once the optimizer has been over the callers.

Reply:
👍